std_msgs/Header header
PoseInMap mapPose
geometry_msgs/Pose originalPose